TotalCrossVM/CMakeLists.txt

```diff
@@ -0,0 +1,620 @@
+cmake_minimum_required(VERSION 3.13.5)
+
+# set the project name
+project(TCVM VERSION 1.0)
```

Comment:
Wouldn't this be 7.0?

Reply:
I don't know 😅, I think we can make this change.